src/DIRAC/RequestManagementSystem/DB/RequestDB.py

```diff
@@ -44,6 +46,15 @@
     distinct,
 )
 
+try:
+    from sqlalchemy.orm import registry
+
+    sqlalchemy_mapper = registry().map_imperatively
+except ImportError:  # registry appeared in sqlalchemy 2.0
```

Comment:
Should this be:
```suggestion
except ImportError:  # registry appeared in sqlalchemy 1.4
```

Reply:
Hum no, it's a new thing

Reply:
If it is then wouldn't it be a problem for v8r0 and integration. Plus:
```
In [1]: import sqlalchemy
   ...: from sqlalchemy.orm import registry
   ...: sqlalchemy.__version__
Out[1]: '1.4.46'
```